docs/devGuide/devGuide.md

```diff
@@ -175,12 +175,42 @@ On Windows:
   You should always check that the generated output is correct before committing any changes to the test sites.
 </box>
 
-##### New features
+#### Adding site content
 
 When adding new features, you should also add new site content into an existing test site or create a new test site to demonstrate the new feature. This is to ensure that your feature can be tested by building that test site.
 
```

Comment:
Add a line before the steps, probably over here, saying
"To add a page to an existing test site, for this example, to `test_site`:"